app/views/resources/show.html.slim

```diff
@@ -1,22 +1,26 @@
 .page-details
   .row.page-details__row
-    .col-xs-4.col-md-4
+    .col-xs-4.col-sm-4
       h6
         span class="glyphicon glyphicon--right glyphicon-#{resource_icon(@resource)}"
-        = @resource.resource_type.capitalize
-    
-    .col-xs-8.col-md-8
+        = @resource.resource_type.upcase
+    .col-xs-12.col-sm-8
       .resource-actions
         = render 'shared/components/btn_count', path: '#', 
                                                 text: 'ADD TO LIST', 
                                                 label: @resource.lists.count,
-                                                classes: 'resource-actions__button--right',
+                                                classes: 'resource-actions__button resource-actions__button--bottom resource-actions__button--right',
                                                 style: 'primary'
-                                                
-
+        - if policy(@resource).edit?
+          = link_to 'EDIT', edit_resource_path(@resource), class: 'btn btn-primary resource-actions__button--right'
+        - if policy(@resource).destroy?
+          = link_to 'DELETE', @resource, method: 'DELETE', class: 'btn btn-default resource-actions__button--right', data: { confirm: 'Are you sure?' }   
   .row.page-details__row.page-details__row--half
     .col-xs-12
-      h1= @resource.title
+      h1.page-details__title
+        - if @resource.priv?
+          span class="glyphicon glyphicon--right glyphicon-eye-close"
```

Comment:
Here's the new version with your recommendations:

![private](https://cloud.githubusercontent.com/assets/2647689/23263775/d81e81ac-fa11-11e6-8cd5-7521e0e50071.png)

I think it looks alright like this, I wanted to use a lock instead of the eye but the one provided in Boostrap glyphicon looks terrible. Could add Font Awesome if you think the lock is better than the eye ;)